http/src/test/resources/reference.conf

```diff
@@ -101,6 +101,7 @@ monitor {
   pollPeriod = 1 second
   maxRetries = -1  # means retry forever
   recreateCluster = true
+  creationTimeLimit = 15 seconds
```

Comment:
very short window for testing so test doesn't take too long. This didn't interfere with any other tests locally, and shouldn't with how our tests are designed, but it's worth noting that this may need to be bumped up in the future.